components/organisms/popup/Popup.tsx

```
@@ -1,144 +1,81 @@
-
+  }, [handleClosePopup]);
   return (
     <div
-      className="fixed inset-0 z-50 bg-black/50 flex items-center justify-center"
-      onClick={handleBackdropClick}
+      className={cn(
+        'fixed inset-0 z-50 bg-black/50 flex items-center justify-center',
+        wrapperClassName
+      )}
+      onClick={handleOutsideClick}
     >
```

Biome (2.4.4)

[error] Static Elements should not be interactive.

(lint/a11y/noStaticElementInteractions)